### PR TITLE
Updating Cce input file to have higher LMax and finer time sampling

### PR DESCRIPTION
**Pull request by keefemitman**
## Proposed changes

Update Cce input file to have higher LMax and finer time sampling after performing BBH convergence tests

### CODE REVIEW

File: tests/InputFiles/Cce/CharacteristicExtract.yaml [hunk 1]
```diff
@@ -63,7 +63,7 @@ Cce:
   StartTime: 0.0
   EndTime: 1000.0
   BoundaryDataFilename: CceR0257.h5
-  H5IsBondiData: false
+  H5IsBondiData: False
```
Comment (knelli2) [inline tests/InputFiles/Cce/CharacteristicExtract.yaml:66]: Why this change?

**Review by knelli2 — CHANGES_REQUESTED** [review]
Can you also update the docs in `CCE.md` to reflect these changes? Also maybe give an explanation that these numbers were chosen based on BBH convergence tests?

File: tests/InputFiles/Cce/CharacteristicExtract.yaml (hunk 1 above)
Comment (keefemitman) [reply tests/InputFiles/Cce/CharacteristicExtract.yaml:66]: Just to be consistent with the other text in the file (not sure why it wasn't that way in the first place)

**Comment by keefemitman** [comment]
> Can you also update the docs in `CCE.md` to reflect these changes? Also maybe give an explanation that these numbers were chosen based on BBH convergence tests?

Will do!

File: tests/InputFiles/Cce/CharacteristicExtract.yaml (hunk 1 above)
Comment (knelli2) [reply tests/InputFiles/Cce/CharacteristicExtract.yaml:66]: Ah ok. Change the ones at the top of the file in the `ResourceInfo` block as well just so the whole file is consistent.

Comment (keefemitman) [reply tests/InputFiles/Cce/CharacteristicExtract.yaml:66]: Done!

**Review by knelli2 — COMMENTED** [review]
Looks good. Can you squash these into one commit and keep the commit name 60 characters or shorter?

**Review by knelli2 — APPROVED** [review]

**Review by nilsdeppe — APPROVED** [review]
